requirements/customer/cli.tex

```diff
@@ -94,7 +94,7 @@ \section{User Client}
 {If set to a string that can be interpreted as RAM amount, informs the server that this amount of RAM is required for the job.}
 \jaoption{threads}{t}
 {If set to an integer, informs the server that this number of threads is required for the job.}
-\jaoption{base}{z}
+\jaoption{image}{i}
```

Comment:
Please move this option so that the options are sorted alphabetically.

Reply:
Hadn't noticed this, should be fixed now.